### DISCUSSION TITLE
Control the internal toolbar actions

### DISCUSSION THREAD

**Discussion by OsamaKhalilMohamed** [KevinVandy/mantine-react-table#332] [Q&A] [answered]
Hello all,

Is there a way to manually control the show/hide columns button that exists in toolbar internal actions? i wanna be able to remove both show all and hide all buttons but keep the reset order or at least attach an event on these buttons so i know which was clicked

thanks

**Comment by KevinVandy** [comment, marked as answer]
Yep. Whole section in the docs for this
